src/components/NavBar/new/MainNavbar/RightMenu.js

```diff
@@ -131,16 +130,31 @@ const RightMenu = ({ mode, onClick }) => {
               </AccordionSummary>
               <AccordionDetails>
                 <Grid container direction="column" spacing={1}>
-                  <Grid item>
-                    <Link to={`/organization`}>
-                      <Grid container spacing={3}>
-                        <Grid item>
-                          <SettingsOutlinedIcon className={classes.orgicon} />
-                        </Grid>
-                        <Grid item>Manage All</Grid>
+                {allowOrgs > 0
```

Comment:
For now multi Organization admins aren't supported yet so what we can do se to take the `current` from redux i.e.

```
  const current = useSelector(
    ({
      org: {
        general: { current }
      }
    }) => current
  );
```

This will give the current organization handle that the user is attached to. So the user can you can just change it to the previoue "Manage All" setting. and redirect to `org/settings/${current}`